airflow_prometheus_exporter/prometheus_exporter.py

```diff
@@ -277,6 +278,57 @@ def get_task_duration_info():
         )
 
 
+def get_landing_times():
+    """Duration of successful tasks in seconds."""
+    with session_scope(Session) as session:
+        max_execution_dt_query = (
+            session.query(
+                DagRun.dag_id,
+                func.max(DagRun.execution_date).label("max_execution_dt"),
```

Comment:
This isn't a dt string right? it is the `execution_date` of the task as datetime. Maybe it would make more sense to call it `max_execution_date`?

Reply:
Again, im just copying code, and following the standard in-file.
But i like you, so i changed it.